react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -21,9 +23,14 @@ const Template = (args) => {
             setProps={(updatedProps) => {
                 setEditedData(updatedProps.editedData);
             }}
+            colorTemplate={colorTemplate}
+            colorTables={colorTables}
         />
     );
 };
 
 export const Default = Template.bind({});
 Default.args = exampleData[0];
+
+export const templateData = Template.bind({});
+templateData.args = { template: colorTemplate, colorTables: colorTables };
```

Comment:
this templateData won't work on storybook as it will need other props from exampleData too

Reply:
I can make necessary changes when I submit separate PR for storybook . If there is no other issues, then may be you can accept this PR.